explorer/src/components/Farming/index.tsx

```diff
@@ -0,0 +1,240 @@
+'use client'
+
+import { EXTERNAL_ROUTES } from '@/constants'
+import Image from 'next/image'
+import { FC, useCallback, useEffect, useMemo, useState } from 'react'
+
+interface ReleaseAsset {
+  name: string
+  browser_download_url: string
+}
+
+interface ReleaseData {
+  assets: ReleaseAsset[]
+}
+
+export const DownloadPage: FC = () => {
+  const [userOS, setUserOS] = useState<string | null>(null)
+  const [releaseAssets, setReleaseAssets] = useState<ReleaseAsset[]>([])
+
+  useEffect(() => {
+    const getOS = () => {
+      const userAgent = window.navigator.userAgent
+      if (userAgent.indexOf('Win') !== -1) return 'Windows'
+      if (userAgent.indexOf('Mac') !== -1) return 'macOS'
+      if (userAgent.indexOf('Linux') !== -1) return 'Linux'
+      return null
+    }
+
+    setUserOS(getOS())
+  }, [])
+
+  useEffect(() => {
+    const fetchLatestRelease = async () => {
+      try {
+        const response = await fetch(EXTERNAL_ROUTES.spaceAcres)
+        if (response.status === 200) {
+          const data: ReleaseData = await response.json()
+          setReleaseAssets(data.assets)
+        }
+      } catch (error) {
+        console.error('Error fetching latest release:', error)
+      }
+    }
+
+    fetchLatestRelease()
+  }, [])
+
+  const getDownloadLink = useCallback(
+    (os: string) => {
+      switch (os) {
+        case 'Windows':
+          return releaseAssets.find((asset) => asset.name.endsWith('.msi'))?.browser_download_url
+        case 'macOS':
+          return releaseAssets.find((asset) => asset.name.endsWith('.dmg'))?.browser_download_url
+        case 'Linux':
+          return releaseAssets.find((asset) => asset.name.endsWith('.deb'))?.browser_download_url
+        default:
+          return '#'
+      }
+    },
+    [releaseAssets],
+  )
+
+  const getAssetName = useCallback(
+    (os: string) => {
+      switch (os) {
+        case 'Windows':
+          return releaseAssets.find((asset) => asset.name.endsWith('.msi'))?.name
+        case 'macOS':
+          return releaseAssets.find((asset) => asset.name.endsWith('.dmg'))?.name
+        case 'Linux':
+          return releaseAssets.find((asset) => asset.name.endsWith('.deb'))?.name
+        default:
+          return ''
+      }
+    },
+    [releaseAssets],
+  )
+
+  const renderDownloadSection = useMemo(() => {
+    const downloadLink = getDownloadLink(userOS || '')
+
+    switch (userOS) {
+      case 'Windows':
+        return (
+          <div className='text-center'>
+            <h2 className='text-2xl font-bold'>Download Space Acres for Windows</h2>
+            <p>Minimum Requirements:</p>
+            <ul className='list-inside list-disc'>
+              <li>OS: Windows 10 or higher</li>
+              <li>Processor: Intel Core i3 or equivalent</li>
+              <li>RAM: 4 GB</li>
+              <li>Disk Space: 500 MB</li>
```

Comment:
The app takes 500 MB, but when you actually start it, with the node it grows to tens of GB. I'd suggest to state 100GB here. 
```suggestion
              <li>Disk Space: 100 GB</li>
```